internal/service/federatedsettingsidentityprovider/data_source_federated_settings_identity_providers.go

```diff
@@ -216,214 +212,26 @@ func PluralDataSource() *schema.Resource {
 }
 func dataSourceMongoDBAtlasFederatedSettingsIdentityProvidersRead(ctx context.Context, d *schema.ResourceData, meta any) diag.Diagnostics {
 	// Get client connection.
-	conn := meta.(*config.MongoDBClient).Atlas
+	connV2 := meta.(*config.MongoDBClient).AtlasV2
 
 	federationSettingsID, federationSettingsIDOk := d.GetOk("federation_settings_id")
 
-	options := &matlas.ListOptions{
-		PageNum:      d.Get("page_num").(int),
-		ItemsPerPage: d.Get("items_per_page").(int),
-	}
-
 	if !federationSettingsIDOk {
 		return diag.FromErr(errors.New("federation_settings_id must be configured"))
 	}
+	// There is no available paginated method
```

Comment:
as of now there is no way to paginate results in the new SDK

Reply:
Although the old SDK defined list options they were **not** used as the options are not present in the v1 API: https://www.mongodb.com/docs/atlas/reference/api-resources-spec/v1/#tag/Federated-Authentication/operation/listIdentityProviders

Given that `page_num` and `items_per_page` are not being used, should we create a followup ticket to deprecate them?

Reply:
Makes sense to me. I can create the ticket